android/src/com/mapswithme/maps/editor/ViralFragment.java

```diff
@@ -72,6 +70,38 @@ public void onClick(View v)
     return builder.setView(root).create();
   }
 
+  private void share()
+  {
+    // TODO add custom sharing in twitter and url to facebook sharing
+    BottomSheetHelper.Builder sheet = BottomSheetHelper.create(getActivity())
+                     .sheet(R.menu.menu_viral_editor)
```

Comment:
Fixed, PTAL

> On Mar 25, 2016, at 7:50 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/ViralFragment.java <https://github.com/mapsme/omim/pull/2541#discussion_r57462677>:
> 
> > @@ -72,6 +70,38 @@ public void onClick(View v)
> >      return builder.setView(root).create();
> >    }
> >  
> > +  private void share()
> > +  {
> > +    // TODO add custom sharing in twitter and url to facebook sharing
> > +    BottomSheetHelper.Builder sheet = BottomSheetHelper.create(getActivity())
> > +                     .sheet(R.menu.menu_viral_editor)
> Странное выравнивание
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2541/files/a197ad50eaf33fe5a3da6c5f99b49ec2c4290ab3#r57462677>